packages/p2p/src/components/verification/__tests__/verification.spec.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { isDesktop } from '@deriv/shared';
+import { useModalManagerContext } from 'Components/modal-manager/modal-manager-context';
+import { useStores } from 'Stores/index';
+import Verification from '../verification';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isDesktop: jest.fn(() => false),
+}));
+
+const mock_modal_manager = {
+    isCurrentModal: false,
+    showModal: jest.fn(),
+    hideModal: jest.fn(),
+};
+
+jest.mock('Components/modal-manager/modal-manager-context', () => ({
+    ...jest.requireActual('Components/modal-manager/modal-manager-context'),
+    useModalManagerContext: jest.fn(() => mock_modal_manager as unknown as ReturnType<typeof useModalManagerContext>),
+}));
+
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn().mockImplementation(() => () => true),
+}));
+
+const mocked_store_values = {
+    is_advertiser: false,
+    nickname: '',
+    poi_status: '',
+    poiStatusText: jest.fn(),
+    toggleNicknamePopup: jest.fn(),
+    props: {
+        history: '',
+    },
+};
+
+describe('<Verification />', () => {
+    it('should render default state', () => {
+        (useStores as jest.Mock).mockReturnValue({
+            general_store: { ...mocked_store_values },
+        });
+
+        render(<Verification should_wrap={false} />);
+
+        const el_dp2p_verification_container = screen.getByTestId('dt_verification_container');
+        expect(el_dp2p_verification_container).toBeInTheDocument();
+    });
+
+    it('Should take default value false for should_wrap when not passed', () => {
+        (useStores as jest.Mock).mockReturnValue({
```

Comment:
this as well, if we already return mocked values above in `jest.mock('Stores', ...)` we can remove this block since we dont change any data from `mocked_store_values`